### PR TITLE
Cleanup: remove en_tutorial_po fixture from store tests

### PR DESCRIPTION
**Pull request by phlax**
This speeds up tests and standardises the fixtures

### CODE REVIEW

File: tests/models/store.py [hunk 1]
```diff
@@ -153,15 +152,14 @@ def test_update_ts_plurals(store_po, test_fs):
...
+def test_update_with_non_ascii(store0, test_fs):
+    store0.state = PARSED
+    orig_units = store0.units.count()
+    with test_fs.open(['data', 'po', 'tutorial', 'en',
                        'tutorial_non_ascii.po']) as f:
         store = getclass(f)(f.read())
-    en_tutorial_po.update(store)
-    assert en_tutorial_po.units[0].target == "Hèllö, wôrld"
+    store0.update(store)
+    assert store0.units[orig_units].target == "Hèllö, wôrld"
```
Comment (dwaynebailey) [inline tests/models/store.py]: I realise it isn't this PR.  But the non-ascii should take things into the unicode range.  I fear that all these are in the first 127 chars so they're still technically single byte ascii, so not doing what is hoped here.

Comment (phlax) [inline tests/models/store.py]: perhaps PR with some better unicode chars?

Comment (phlax) [inline tests/models/store.py]: ☠

**Comment by phlax** [comment]
closed in https://github.com/translate/pootle/commit/d5a3eecbc01dc5c44932d1227e245db56cc1d6d1

**Pull request closed without merging**